src/components/episodeUtils.ts

```diff
@@ -0,0 +1,35 @@
+import { EpisodeInfo } from "./Episode";
+
```

Comment:
Consider adding a folder called utils with separate function files. The reason for that is that this file feels overloaded with information and function that do completely different things. 
But note - It's good that you created separate functions for it.